### DISCUSSION TITLE
My public repository cannot be found from google search

### DISCUSSION THREAD

**Discussion by croquies** [community/community#33777] [Repositories] [answered]
Few months ago I created a "private" repository, but I changed it to "public" few weeks later.
But I can not find this repository from google search engine.
What can I do make this repo on listed for google search results?
The repository is : https://github.com/fika-dev/fika-cli

Thanks!!

**Comment by jfullstackdev** [comment, marked as answer]
I had the same thing in the past, to jump start I started using Google Analytics and Google Search Console,

other than that is to keep your links being shared in other platforms, that way the web crawler will recognize and remember your links and it will be made available at the searches

**Reply by croquies** [reply]
Thank you for sharing your experience!
But google search console requires to verify I owns the domain.
It seems possible if it is github pages, but It seems not possible to me.

**Reply by jfullstackdev** [reply]
if it's a converted to site, using the `.github.io`, then you can own it, as far as I can remember, I uploaded a file for that verification in the same repo but the one being validated is the hosted site, not the repo site, at least, if you are promoting your sites, visitors may come from your site first then will be redirected going to your repo 

and again, it helps the web crawler remember your site

**Reply by croquies** [reply]
Thank you for kind suggestion!!
